.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Attach Debugger",
+            "type": "node",
+            "request": "attach",
+            "port": 5858,
+            "address": "localhost",
+            "restart": true,
+            "sourceMaps": false,
+            "outFiles": [],
+            "localRoot": "${workspaceRoot}",
+            "remoteRoot": null
+        }
+    ]
+}
```

Comment:
As we run with node foreman, and the nodemon - we can't easily run it as a launchable process in VS Code, however we can successfully connect and reconnect to a running node process, which works fine 👍